tests/peer/meshRoom.js

```diff
@@ -371,6 +383,94 @@ describe('MeshRoom', () => {
 
       meshRoom.send(data);
     });
+
+    describe('when the data type is string', () => {
```

Comment:
It seems better to add a normal test each type (string, buffer, object). Like 19MB can be sent.